test/integration/setup/loadContracts.ts

```diff
@@ -1,9 +1,9 @@
-import mainnetAddresses from '@protocol/mainnetAddresses';
+import { MainnetContractsConfigs } from '@protocol/mainnetAddresses';
 import { ethers } from 'hardhat';
 import { MainnetContracts, MainnetContractsConfig, NamedAddresses } from '@custom-types/types';
 
 export async function getAllContracts(): Promise<MainnetContracts> {
-  const addresses = mainnetAddresses as MainnetContractsConfig;
+  const addresses = MainnetContractsConfigs as MainnetContractsConfig;
```

Comment:
case is a bit confusing because the object and the type are both CamelCase with only a "s" difference

Reply:
Good callout, I'll make this more clear.

Reply:
Removed MainnetContratsConfig interface and now just using the type "MainnetContractsType"